Source/Core/DolphinWX/Config/GeneralConfigPane.cpp

```diff
@@ -23,13 +23,13 @@
 GeneralConfigPane::GeneralConfigPane(wxWindow* parent, wxWindowID id) : wxPanel(parent, id)
 {
   m_cpu_cores = {
-      {PowerPC::CORE_INTERPRETER, _("Interpreter (slowest)")},
-      {PowerPC::CORE_CACHEDINTERPRETER, _("Cached Interpreter (slower)")},
+      {PowerPC::CORE_INTERPRETER, _("Interpreter (Slowest)")},
+      {PowerPC::CORE_CACHEDINTERPRETER, _("Cached Interpreter (Slower)")},
 #ifdef _M_X86_64
-      {PowerPC::CORE_JIT64, _("JIT Recompiler (recommended)")},
-      {PowerPC::CORE_JITIL64, _("JITIL Recompiler (slow, experimental)")},
+      {PowerPC::CORE_JIT64, _("JIT Recompiler (Recommended)")},
+      {PowerPC::CORE_JITIL64, _("JITIL Recompiler (Slow, Experimental)")},
 #elif defined(_M_ARM_64)
-      {PowerPC::CORE_JITARM64, _("JIT Arm64 (experimental)")},
+      {PowerPC::CORE_JITARM64, _("JIT Arm64 (Experimental)")},
```

Comment:
Using a tooltip is a bad idea, because it's very hard to discover. The text is short enough that putting it right next to the title like we do right now doesn't lead to any practical problems.